server/build.go

```diff
@@ -269,3 +286,26 @@ func cancelBuild(c *gin.Context) {
 
 	c.JSON(http.StatusOK, BuildResp)
 }
+
+// Int32 converts int32 to *int32
+func Int32(v int32) *int32 { return &v }
```

Comment:
I might be missing it but is this used anywhere?

I can't seem to find where it's used so I believe this should be removed.

Reply:
Removed in the latest commit.